API.md

```diff
@@ -699,7 +708,7 @@ schema.validate(3);     // returns err.message === 'Was REALLY expecting a strin
 
 Generates a schema object that matches an array data type. Note that undefined values inside arrays are not allowed by default but can be by using `sparse()`.
 
-Supports the same methods of the [`any()`](#any) type.
+#### Every [`any()`](#any) methods
```

Comment:
I'm looking for a way to explicit that array & such do support every `any()` methods.

The existing `Supports the same methods...` is blur in other existing text and not stated clearly. Plus, it does not pop in the link listing at the top of the page.

So something like (in the top links):
```
— array
   — every any() methods
```

And (in the page):
```
— array
(title) every any() methods
(title) sparse
(title) ...
```